odoo/tests/common.py

```diff
@@ -822,12 +822,12 @@ def setUp(self):
         self.opener = requests.Session()
         self.opener.cookies['session_id'] = self.session_id
 
-    def url_open(self, url, data=None, timeout=10):
+    def url_open(self, url, data=None, timeout=10, headers=None):
         if url.startswith('/'):
             url = "http://%s:%s%s" % (HOST, PORT, url)
         if data:
             return self.opener.post(url, data=data, timeout=timeout)
```

Comment:
If we are allowed to add this `headers` parameter, then it should forwarded to the `post` request too.